### ISSUE TITLE
What is the difference between postnet and CBHG?

### ISSUE THREAD

**Issue by Approximetal** [xcmyz/FastSpeech#85]
I saw [history updates](https://github.com/xcmyz/FastSpeech/commit/1f5aa2a33be2e7fab3a9c2fb8666a08c1240bdcc#diff-93ff136fae812392eb0f68d1ce89b7fe) replace the Postnet using CBHG. Is there any speed or performance enhancement after changing the structure?

**Comment by xcmyz** [comment]
> I saw [history updates](https://github.com/xcmyz/FastSpeech/commit/1f5aa2a33be2e7fab3a9c2fb8666a08c1240bdcc#diff-93ff136fae812392eb0f68d1ce89b7fe) replace the Postnet using CBHG. Is there any speed or performance enhancement after changing the structure?

postnet in fastspeech is not necessary

**Comment by Approximetal** [comment]
In my knowledge, postnet is added to enhance the performance of mel output, is CBHG plays the same role? What about remove this module?

**Comment by xcmyz** [comment]
> In my knowledge, postnet is added to enhance the performance of mel output, is CBHG plays the same role? What about remove this module?

yes, cbhg is the part of postnet and it is not necessary for fastspeech to add postnet in network.
